### ISSUE TITLE
deal with canonicalisation change

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#1741]
From IRC:
```
lfam: Hm, I think the SSH relative path changes in 1.0.8rc1 trigger a relocated repository warning
ThomasWaldmann$ lfam: yes, it is due to a change of canonicalisation
ThomasWaldmann$ old: relpath -> /~/relpath, new: relpath -> /./relpath
lfam$ It breaks automation until the user confirms the new location. But it's okay if that's expected
ThomasWaldmann$ guess a better visible changelog entry is needed
```

**Comment by enkore** [comment]
Can we fix this so that it doesn't say that two actually identical paths are not identical?

**Comment by ThomasWaldmann** [comment]
. is relative and on the server, so only the server after connection is established knows what it is (outside influence via ssh forced command with "cd ..."). the warning comes from the client.

**Comment by ThomasWaldmann** [comment]
See PR #1742.

**Comment by ThomasWaldmann** [comment]
fixed by #1742.
